packages/flutter/lib/src/foundation/isolates.dart

```diff
@@ -16,26 +16,15 @@ import '_isolates_io.dart'
 /// {@macro flutter.foundation.compute.callback}
```

Comment:
I believe this reference and the one below point to deleted macros.

Reply:
Thanks. It makes now sense to me, a `@macro` is basically copying in code from the `@template`.
Done.